spec/03-plugins/24-response-rate-limiting/04-access_spec.lua

```diff
@@ -90,787 +110,842 @@ end
 
 
 for _, strategy in helpers.each_strategy() do
-for _, policy in ipairs({"local", "cluster", "redis"}) do
-
-describe(fmt("#flaky Plugin: response-ratelimiting (access) with policy: #%s [#%s]", policy, strategy), function()
-
-  lazy_setup(function()
-    local bp = init_db(strategy, policy)
+  for _, policy in ipairs({"local", "cluster", "redis"}) do
 
-    if policy == "local" then
-      SLEEP_TIME = 0.001
-    else
-      SLEEP_TIME = 0.15
-    end
-
-    local consumer1 = bp.consumers:insert {custom_id = "provider_123"}
-    bp.keyauth_credentials:insert {
-      key      = "apikey123",
-      consumer = { id = consumer1.id },
-    }
-
-    local consumer2 = bp.consumers:insert {custom_id = "provider_124"}
-    bp.keyauth_credentials:insert {
-      key      = "apikey124",
-      consumer = { id = consumer2.id },
-    }
-
-    local route1 = bp.routes:insert {
-      hosts      = { "test1.com" },
-      protocols  = { "http", "https" },
-    }
-
-    bp.response_ratelimiting_plugins:insert({
-      route = { id = route1.id },
-      config   = {
-        fault_tolerant = false,
-        policy         = policy,
-        redis_host     = REDIS_HOST,
-        redis_port     = REDIS_PORT,
-        redis_password = REDIS_PASSWORD,
-        redis_database = REDIS_DATABASE,
-        limits         = { video = { second = ITERATIONS } },
-      },
-    })
+    for redis_conf_name, redis_conf in pairs(redis_confs) do
```

Comment:
I know these were copied from the rate-limiting. I think we can add some test cases only for the SSL connection feature instead of doing a loop for every existing test. This will reduce running time.